.travis.yml

```diff
@@ -15,5 +15,15 @@ before_install:
   - make test-image deps
   - sleep 30
```

Comment:
I added this when I was trying to fix the end-to-end tests but forgot to remove when I cleaned up.